cuda_core/tests/cython/build_tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+SCRIPTPATH=$(dirname $(realpath "$0"))
+CPLUS_INCLUDE_PATH=$SCRIPTPATH/../../cuda/core/experimental/include:$CUDA_HOME/include:$CPLUS_INCLUDE_PATH cythonize -3 -i $(dirname "$0")/test_*.pyx
```

Comment:
Note: This approach left a bug uncaught (fixed in commit 37a6850): Ideally, we want to have a cmdline functionality like PyBind11 that returns the header path, something like `python -m cuda.core --include-path`. Here, we use the header from the local clone of this repo, meaning we do not check if the header is actually packaged (and it was not before the fix). However, I think the said functionality needs to be done in a separate PR...